cypress/integration/login_spec.js

```diff
@@ -0,0 +1,70 @@
+
+const {port, hostName} = require('../../config/env/all')
+describe('/login behaviour', () => {
+  
+  beforeEach(function () {
+    cy.visit(`http://${hostName}:${port}/login`)
+  })
+
+  it('should have tutorial Guide link', () => {
+    cy.get("a[href='/tutorial']" )
+    .should('have.attr', 'target', '_blank') 
+    .and('be.visible')
+  })
+
+  it('Should open the tutorial in another tab', () => {
+    cy.get("a[href='/tutorial']" ).then(function($a){
+      const href = 
+      $a.prop('href')
+      cy.visit(href)
+      cy.url().should('include', 'tutorial')
+    })
+  })
+
+
+  it('should have admin user able to login', () => {
+    cy.get("#userName").type("admin")
```

Comment:
probably best to put these out in a constant file, or even allow overriding them from an env variable but not required.

Reply:
Yeah! I will refactor that. Thanks for the feedback! 👍